src/Components/FilterSection/FilterSection.jsx

```diff
@@ -0,0 +1,216 @@
+                            checked={filters.RATING === 2}
+                        />
+                        2 star & above
+                    </label>
+                    <label>
+                        <input
+                            type="radio"
+                            name="rating-input"
+                            onChange={(e) =>
+                                productDispatchFun({ type: "RATING", payload: { value: Number(e.target.value) } })
+                            }
+                            value="1"
+                            checked={filters.RATING === 1}
+                        />
+                        1 star & above
+                    </label>
+                </div>
+                <hr></hr>
+
+                <div className="filter-content">
+                    <h4 className="filter-header">Categories</h4>
+
+                    <label>
+                        <input
+                            type="checkbox"
+                            onChange={(e) =>
+                                productDispatchFun({
+                                    type: "CATEGORY",
+                                    payload: { category: "GOGGLES", value: e.target.checked },
+                                })
+                            }
+                            checked={filters.categories.GOGGLES}
+                        />
+                        Goggles
+                    </label>
+                    <label>
+                        <input
+                            type="checkbox"
+                            onChange={(e) =>
+                                productDispatchFun({
+                                    type: "CATEGORY",
+                                    payload: { category: "STICKERS", value: e.target.checked },
+                                })
+                            }
+                            checked={filters.categories.STICKERS}
+                        />
+                        Stickers
+                    </label>
+                    <label>
+                        <input
+                            type="checkbox"
+                            onChange={(e) =>
+                                productDispatchFun({
+                                    type: "CATEGORY",
+                                    payload: { category: "TAILTIDY", value: e.target.checked },
+                                })
+                            }
+                            checked={filters.categories.TAILTIDY}
+                        />
+                        Tail Tidy
+                    </label>
+                    <label>
+                        <input
+                            type="checkbox"
+                            onChange={(e) =>
+                                productDispatchFun({
+                                    type: "CATEGORY",
+                                    payload: { category: "HANDGUARDS", value: e.target.checked },
```

Comment:
you can make a constants file for all these action types

Reply:
made the changes, thanks for the feedback!